examples/everything.amp.html

```diff
@@ -136,7 +136,10 @@ <h1 id="top">AMP #0</h1>
     <a class="goto" href="#img0">Image0</a>
     <a class="goto" href="#footer">Footer</a>
   </p>
-
+  
+  <!--Add to test flex layout-->
+  <amp- src="https://lh3.googleusercontent.com/5rcQ32ml8E5ONp9f9-Rf78IofLb9QjS5_0mqsY1zEFc=w400-h300-no-n" width=400 height=300 layout=flex></amp-img>
```

Comment:
Without `display: flex` parent this image should not display. I think that's what you are running into. Please go over https://css-tricks.com/snippets/css/a-guide-to-flexbox/ - there are some good examples here.